frontend/src/Buildings/buildingReducer.js

```diff
@@ -13,11 +16,15 @@ const initialState = {
 export default function buildings(state = initialState, action) {
   switch (action.type) {
     case FETCH_BUILDINGS_REQUEST:
+    case ADD_BUILDING_REQUEST:
```

Comment:
this would never happen because right now you don't have an action for this in the actionCreator